src/pybind/ctc/ctc.mk

```diff
@@ -0,0 +1,29 @@
+
+WGET ?= wget
+
+# This commit is the latest commit available as of 2019.01.15 .
+# Update it if needed.
+COMMIT_ID := bc29dcfff07ced1c7a19a4ecee48e5ad583cef8e
```

Comment:
About this CTC stuff, can we talk about what the options were here?  I see that Warp-CTC has PyTorch bindings already.  What are the advantages of loading it via the Kaldi module?

Reply:
I can think of the following advantages:

(1) It does not depend on PyTorch and can be used for any version of PyTorch.

(2) If kaldi pybind switches to other frameworks, we can still use warp-ctc,
as long as the framework implements the DLPack interface.

(3) The users are not required to install warp-ctc PyTorch bindings by themselves,
which will eliminate lots of issues encountered during the installation (see https://github.com/SeanNaren/warp-ctc/issues).

It is straightforward to wrap warp-ctc and why not to wrap it in kaldi pybind?

Reply:
OK cool.